src/Shared/Microsoft.AspNetCore.Razor.Test.Common/TestCompilation.cs

```diff
@@ -60,7 +60,7 @@ private static IEnumerable<string> ResolvePaths(CompilationLibrary library)
     public static string AssemblyName => "TestAssembly";
 
     // When we use Basic.Reference.Assemblies everywhere, this overload should be removed.
-    // Tracked by https://github.com/dotnet/razor/issues/10343.
+    // See also https://github.com/dotnet/razor/issues/10343.
     public static CSharpCompilation Create(Assembly assembly, SyntaxTree syntaxTree = null)
```

Comment:
This is still used by Version1_X and Version2_X of the Mvc tests. I don't think I can easily remove that because I would need Basic.Reference.Assemblies for AspNetCore 1.0 and 2.0 which might be difficult to create.

Reply:
That doesn't mean we can't try to nerd snipe @jaredpar with it.

Reply:
The official reference assemblies only go back to 3.0.

https://www.nuget.org/packages/Microsoft.AspNetCore.App.Ref/

Do we know where the 1.0 / 2.0 ref assemblies are? If there is an official copy somewhere could consider taking an approach like we did with `netstandard1.3`? Basically just create the correct set on disk, check them in as resources and build a basci ref pack for that.

Reply:
Well there are many Microsoft.AspNetCore.* package stuck at version 2.2.0, e.g., https://www.nuget.org/packages/Microsoft.AspNetCore.Mvc. They are the full dlls, not just ref dlls, I'm not sure if ref dlls were published at the time.